src/docs/truffle/reference/.pages

```diff
@@ -0,0 +1,4 @@
+nav:
+  - Command line options: truffle-commands.md
```

Comment:
Nav title "Command line options" is different from that in the page (`/docs/truffle/reference/truffle-commands/`) which is "Truffle Commands"